code/modules/medical/surgery_tools.dm

```diff
@@ -358,6 +358,7 @@ CONTAINS:
 	icon_state = "defib-on"
 	item_state = "defib"
 	pickup_sfx = "sound/items/pickup_defib.ogg"
+	w_class = W_CLASS_TINY
```

Comment:
A `w_class` isn't necessary. If not specified it defaults to its parent (`/obj/item`, in this case, which has a default `w_class` of `W_CLASS_NORMAL`, as seen [here](https://github.com/goonstation/goonstation/blob/6950eb795b076ffe43ba12e0a0cee9bebfdae5a7/code/obj/item.dm#L61)).

Medical belts specifically allow defibs, as seen [here](https://github.com/goonstation/goonstation/blob/55db17e3be6bd680c632ecb759e34d9c230e8063/code/obj/item/storage/backpack_belt_etc.dm#L480).

Reducing the size to tiny would allow them to fit in pockets, in boxes, and in all belts (rather than just medical belts).